preview-src/asciidoc/images.adoc

```diff
@@ -0,0 +1,116 @@
+In AsciiDoc, creating paragraphs is a straightforward process that does not require any special markup. A paragraph can be defined as one or more lines of consecutive text that are logically grouped together. To differentiate between paragraphs, you simply need to insert at least one blank line between them.
+--
+
+[.float-group]
+--
+image::/img/multirepo-ssg.svg[Multirepo SSG,300,float=left,role=float-gap]
+In AsciiDoc, creating paragraphs is a straightforward process that does not require any special markup. A paragraph can be defined as one or more lines of consecutive text that are logically grouped together. To differentiate between paragraphs, you simply need to insert at least one blank line between them.
+--
+
+== Inline images
+
+Click image:/img/play_circle_FILL0_wght400_GRAD0_opsz24.svg[title=Play] to get the party started.
+
+Click image:/img/pause_circle_FILL0_wght400_GRAD0_opsz24.svg[title=Pause] when you need a break.
+
+== Image theming (light/dark mode)
+
+[NOTE]
+======
+You can define an image such that it appears differently in light mode vs. dark mode.
+There are methods for doing this.
+
+Method 1: Two separate images::
+Define two images, one that looks good in light mode and one that looks good in dark mode.
+Assign the corresponding roles -- `for-light` and `for-dark` -- to each image.
++
+All non-SVG images, such as screenshot PNG files, must use this method.
+All SVG images that do not use CSS variables must also use this method.
++
+.Method 1 -- block image
+[source,asciidoc]
+----
+image::light-mode-illustration.png[Alt text,400,role=for-light]
+image::dark-mode-illustration.png[Alt text,400,role=for-dark]
+----
++
+.Method 1 -- inline image
+[source,asciidoc]
+----
+Click the Astra Vector icon image:/img/astra-vector-light.svg[title="Astra Vector",role=for-light] image:/img/astra-vector-dark.svg[title="Astra Vector",role=for-dark].
+----
+
+Method 2: SVG image with CSS variables::
+Use the custom `svg` macro to define an SVG image that uses CSS variables to support light and dark mode.
++
+.Method 2 -- block image
+[source,asciidoc]
+----
+svg::ROOT:illustration.svg[Alt text,400]
+----
++
+.Method 2 -- inline image
+[source,asciidoc]
+----
+Click the Astra Vector icon svg:ROOT:your-diagram.svg[title="Astra Vector"].
+----
++
+[IMPORTANT]
+====
+CSS variables must use the tokens supported by the UI.
+For example:
+
+[source,svg]
+----
+<svg viewBox="0 0 300 200" fill="none">
+  <path fill="var(--ds-text-primary)" d="..." />
+  <path stroke="var(--ds-primary-outlined-border)" d="..." />
+  <path fill="var(--ds-neutral-outlined-border)" d="..." />
+</svg>
+----
+====
+======
+
+image::/img/light-mode-illustration.png[Alt text,400,role=for-light]
+image::/img/dark-mode-illustration.png[Alt text,400,role=for-dark]
+
+Click the Astra Vector icon image:/img/astra-vector-light.svg[title="Astra Vector",role=for-light] image:/img/astra-vector-dark.svg[title="Astra Vector",role=for-dark].
```

Comment:
I've added examples that utilize Method 1, but I didn't have any ready-made examples that use CSS variables. Do you have any available to add?

Reply:
Since the svg macro doesn't work in the UI preview I think the current description will have to suffice